velocity-engine-core/src/main/java/org/apache/velocity/app/event/EventCartridge.java

```diff
@@ -267,54 +265,49 @@ public final boolean attachToContext( Context context )
      * @param rs
      * @since 1.5
      */
-    public void initialize (RuntimeServices rs)
+    public void initialize( RuntimeServices rs )
     {
 
-        for ( Iterator i = referenceHandlers.iterator(); i.hasNext(); )
+        for ( Iterator<ReferenceInsertionEventHandler> i = referenceHandlers.iterator(); i.hasNext(); )
```

Comment:
These `for` loops could be changed to the enhanced form: `for (ReferenceInsertionEventHandler eh : referenceHandlers)`